kinto_http/__init__.py

```diff
@@ -862,6 +863,18 @@ def delete_records(self, *, collection=None, bucket=None, safe=True, if_match=No
         resp, _ = self.session.request("delete", endpoint, headers=headers)
         return resp["data"]
 
+    def get_history(self, *, bucket=None, **kwargs):
+        endpoint = self.get_endpoint("history", bucket=bucket)
```

Comment:
In the previous round of this PR, you suggested it would be good to check for the `history` plugin.

Reply:
Yeah, I thought we had the capabilities check already, but I realized we don't. So I went for the shortest approach...